lib/fetch/request.js

```diff
@@ -35,6 +35,12 @@ const { getMaxListeners, setMaxListeners, getEventListeners, defaultMaxListeners
 const kAbortController = Symbol('abortController')
 
 const requestFinalizer = new FinalizationRegistry(({ signal, abort }) => {
+  // Currently FinalizationRegistry has a problem and will explicitly call unregister.
+  // https://github.com/nodejs/node/issues/49344
+  // https://github.com/nodejs/node/issues/47748
+  // It will be removed in the future.
+  // The unregister key is abort.
+  requestFinalizer.unregister(abort)
```

Comment:
you don't need to call unregister here. The finalizationregistry already fired.

Reply:
Thank you for the notice.